Block/example.jsx

```diff
@@ -31,11 +31,23 @@ export default {
 
       'Left-Aligned': <Block.Installments.Main leftAlign>
         <Block.Installments.Content>
-          <Paragraph>
+          <Paragraph.Primary>
             Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris at aliquet leo.
-          </Paragraph>
+          </Paragraph.Primary>
         </Block.Installments.Content>
-      </Block.Installments.Main>
+      </Block.Installments.Main>,
+
+      'Bordered': <Block.Bordered>
+        <Paragraph.Secondary>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris at aliquet leo.
+        </Paragraph.Secondary>
+      </Block.Bordered>,
+
+      'Custom-bordered': <Block.Bordered style={{borderRadius: 0}}>
```

Comment:
Well, don't we? We need to handle the `customize` prop for when we are wrapped by the `Theme` component?